### PR TITLE
Parse volume attachment/detachment messages from fog responses

### PR DESCRIPTION
**Pull request by mansam**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1530345

image

### CODE REVIEW

File: app/models/manageiq/providers/openstack/cloud_manager/cloud_volume/operations.rb [hunk 1]
```diff
@@ -18,6 +18,10 @@ def raw_attach_volume(server_ems_ref, device = nil)
         service.servers.get(server_ems_ref).attach_volume(ems_ref, device)
       end
     end
+  rescue => e
+    volume_name = name.empty? ? ems_ref : name
```
Comment (aufi) [inline app/models/manageiq/providers/openstack/cloud_manager/cloud_volume/operations.rb]: Wouldn't be better use ```name.blank?``` to not fail when _name_ would be _nil_ or nil cannot appear there?

Comment (mansam) [reply app/models/manageiq/providers/openstack/cloud_manager/cloud_volume/operations.rb]: I think you're right that `blank?` is the better choice here.

Comment (mansam) [reply app/models/manageiq/providers/openstack/cloud_manager/cloud_volume/operations.rb]: Ended up using "presence" because rubocop complained :)

**Comment by aufi** [comment]
Looks good to me, merging :+1:

**Pull request merged**